### DISCUSSION TITLE
Any way to communicate ORP and pH back to intellicenter?

### DISCUSSION THREAD

**Discussion by Telemachus1977** [tagyoureit/nodejs-poolController#1051] [RS485]
I've noticed with my buddies intellicenter that there is a spot where it can report the pH and the ORP. 

He would like to set up a NJS-PC/REM chemical controller, but would like to have that information relayed back through the app. 

Is there any way to have it feed this information back into the pentair system to report on their app?

**Comment by tagyoureit** [comment]
This isn't the only controller, but the short answer is no.  You need an Intellichem for the OCP to correctly process this information.
